files/usr/sbin/refresh_initrd

```diff
@@ -14,28 +14,31 @@ refresh=no
 test /etc/sysconfig/clock  -nt $initrd && refresh=yes
 test /etc/sysconfig/kernel -nt $initrd && refresh=yes
 
-while read module ; do
+lsinitrd /boot/initrd | grep -E "$modules/.*\.ko" | while read module ; do
```

Comment:
this runs the read in a subshell so setting refresh later won't have a effect in the actual script, no?

Reply:
Yep good catch ... that is the advantage of using shells like bash and ksh, that is not having a subshell

Reply:
Makes sense, thanks!

Reply:
@lnussel @bitstreamout please take a look, I think I've fixed the review comment. However, cannot say it's done gracefully... 

A while ago I suggested to create a sub-package of aaa_base that would include GLPv3-free stuff. Maybe it's better solution than this horrible refactoring?